### PR TITLE
Dev/billing

### PR DESCRIPTION
**Pull request by CallumEddisford**
Resolves: https://github.com/xedi/sdk-js/issues/10

### CODE REVIEW

File: src/Services/Billing/InvoiceItems.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Service from "../Service";
+
+class InvoiceItems extends Service {
+}
```
Comment (Smudge3806) [inline src/Services/Billing/InvoiceItems.ts:4]: Is this needed rn

File: src/Services/Billing/Subscriptions.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import Service from "../Service";
+import JsonResponse from "../../Interfaces/JsonResponse";
+import {Subscription} from "../../Models/Models";
+import {AxiosResponse} from "axios";
+
+class Subscriptions extends Service {
+
+    /**
+     * Check if current business is subscribed to entity
+     * TODO@CE: @CS what do you reckon we do for "boolean" responses
```
Comment (Smudge3806) [inline src/Services/Billing/Subscriptions.ts:10]: You'll still have to wait for the round trip so I would possibly not care about the response and just go with the standard `then/catch` for pass or fail behaviour

File: src/Services/Billing/Taxes.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import Service from "../Service";
+
+class Taxes extends Service {
```
Comment (Smudge3806) [inline src/Services/Billing/Taxes.ts:3]: is this needed rn

File: src/Services/Billing/Subscriptions.ts (hunk 2 above)
Comment (CallumEddisford) [reply src/Services/Billing/Subscriptions.ts:10]: Yeah what i did actually, forgot to removed these :joy:

**Pull request merged**